src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java

```diff
@@ -0,0 +1,145 @@
+package org.testcontainers.containers;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Testcontainers
+public class TarantoolCartridgeMigrationsTest {
+
+    @Container
+    private static final TarantoolCartridgeContainer container =
+            new TarantoolCartridgeContainer(
+                    "Dockerfile",
+                    "cartridge/instances.yml",
+                    "cartridge/topology.lua")
+                    .withDirectoryBinding("cartridge")
+                    .withStartupTimeout(Duration.ofSeconds(300))
+                    .withLogConsumer(new Slf4jLogConsumer(
+                            LoggerFactory.getLogger(TarantoolCartridgeStaticContainerTest.class)));
+
+    @BeforeAll
+    public static void start_container(){
+        if (!container.isRunning()) {
+            try {
+                container.start();
+            } catch (Exception e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
+            }
+        }
+
+    }
+
+    @Test
+    public void test_migrations_migrator() throws Exception {
+
+        if(container.isRunning()){
+            try {
+                List<?> resultMigration = container.executeCommand("require('migrator').up()").get();
+            } catch (Exception e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
+    @Test
+    public void test_migrations_curl(){
```

Comment:
Do not use curl please to reduce dependencies on external modules

Reply:
I don't understand.  Only standart container tools are user here